internal/acceptance/recipient_test.go

```diff
@@ -43,3 +43,57 @@ func TestUcAccCreateRecipientDb2DbAws(t *testing.T) {
 		}`,
 	})
 }
+
+func TestUcAccUpdateRecipientDb2Open(t *testing.T) {
+	unityWorkspaceLevel(t, step{
+		Template: `
+		resource "databricks_recipient" "db2open" {
```

Comment:
ditto: let's refactor this to make it more obvious what the actual changes between each step.